README.md

```diff
@@ -183,11 +205,11 @@ Locally:
 1. Fix all files in the project, `Warning: This writes to your files!` `cd docs`
    `markdownlint './docs/**/*.+(md|mdx)' --ignore node_modules --fix`
 
-## How-To
+## How to...
 
-### Add content to example page
+### Add content to Examples page
```

Comment:
I don't want to go through each caption here but since you changed this one, what do you think about changing it to this:

```suggestion
### Add content to the "examples" page
```